internal/natsclient/jetstream.go

```diff
@@ -65,32 +73,48 @@ func (j *Jetstream) createJetstreamContext() {
 }
 
 func (j *Jetstream) createStream() {
```

Comment:
change the name to indicate we wanna update stream in first place

Reply:
Fixed. 👍